crates/swift-bridge-ir/src/codegen/generate_rust_tokens/shared_enum.rs

```diff
@@ -130,14 +130,31 @@ impl SwiftBridgeModule {
             convert_ffi_variants_to_rust.push(convert_ffi_variant_to_rust);
         }
 
-        // TODO:
-        //  Parse any derives that the user has specified and combine those with our auto derives.
-        let automatic_derives = if shared_enum.has_one_or_more_variants_with_data() {
+        // Auto derives
+        let mut derives = if shared_enum.has_one_or_more_variants_with_data() {
             vec![]
         } else {
             vec![quote! {Copy}, quote! {Clone}]
         };
 
+        // User derives
+        let mut derive_impls = vec![];
```

Comment:
Let's change the variable name since reads as if it is storing `impl DerivedTrait for for MyEnum { .. }`.

Maybe call it `derive_impl_ffi_bridges`? If you have a better name in mind, go for it.

Reply:
[`5aba738` (#194)](https://github.com/chinedufn/swift-bridge/pull/194/commits/5aba738af46992d65943ad6ec77d564069decffe)